src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -1004,7 +1004,8 @@ grpc_subchannel_get_connected_subchannel(grpc_subchannel* c) {
 void* grpc_connected_subchannel_call_get_parent_data(
     grpc_subchannel_call* subchannel_call) {
   grpc_channel_stack* chanstk = subchannel_call->connection->channel_stack();
-  return (char*)subchannel_call + sizeof(grpc_subchannel_call) +
+  return (char*)subchannel_call +
+         GPR_ROUND_UP_TO_ALIGNMENT_SIZE(sizeof(grpc_subchannel_call)) +
          chanstk->call_stack_size;
```

Comment:
I think we need to round up the call stack size too.

Basically, any time we allocate space for more than one object in a single allocation, we need to round up the sizes of all except the last object.